src/components/movie-tile/movie-tile.jsx

```diff
@@ -15,9 +15,11 @@ const MovieTile = ({ movie, onTileClick }) => {
     setIsMenuOpened(false);
   };
 
+  const getYear = (releaseDate) => new Date(releaseDate).getFullYear();
```

Comment:
Might move it out as a util function, since it's used in multiple components

Reply:
Moved out getYear to the utils file.